resolvers.js

```diff
@@ -255,6 +264,25 @@ const Resolvers = {
 
     uploadDateTime(ds) {
       return ds._source.ds_upload_dt;
+    },
+
+    fdrCounts(ds, {inpFdrLvls}) {
+      let outFdrLvls = [], outFdrCounts = [];
+      if(ds._source.annotation_counts && ds._source.ds_status === 'FINISHED') {
+        let inpAllLvlCounts = ds._source.annotation_counts[0].counts;
+        inpFdrLvls.forEach(lvl => {
+          inpAllLvlCounts.find(lvlCount => {
```

Comment:
It's a bit confusing usage of `find()`. The callback function should just check that `lvlCount.level === lvl`. Then, `find()` would return the corresponding count, and you extract necessary data from it. Keep in mind that `find()` may return `undefined`.